src/components/clusterWizard/wizardTransition.ts

```diff
@@ -167,14 +167,17 @@ export const getWizardStepHostStatus = (
 ): Host['status'] => {
   const { status } = host;
   if (['insufficient', 'pending-for-input'].includes(host.status)) {
-    const validationsInfo = stringToJSON<HostValidationsInfo>(host.validationsInfo) || {};
+    const validationsInfo = stringToJSON<HostValidationsInfo>(host.validationsInfo);
+    if (!validationsInfo) {
```

Comment:
This is not necessary as `checkHostValidationGroups` and `checkHostValidations` will return false if `validationsInfo` is empty object. See wizardTransitions.test.ts